### PR TITLE
Add ``adduser`` command for Zerqu with the power of Flask-Script

### PR DESCRIPTION
**Pull request by kxxoling**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from zerqu.app import create_app
+from zerqu.models.base import db
+from zerqu.models.user import User
+from zerqu.libs import cache
+
+
+CONFIG = os.path.abspath('./local_config.py')
+
+app = create_app(CONFIG)
+db.init_app(app)
```
Comment (lepture) [inline manage.py]: 不需要， create_app 里有。

**Comment by kxxoling** [comment]
@lepture ``create_app`` doesn't have ``db.init_app`` actually, but ``register_base`` do. Should I replace it with ``zerqu.register_base``?

**Comment by lepture** [comment]
@kxxoling from zerqu import create_app

**Comment by kxxoling** [comment]
@lepture OK...

**Comment by kxxoling** [comment]
@lepture Updated.

**Pull request merged**